src/components/Sample/Sample.tsx

```diff
@@ -2,9 +2,10 @@ import React, { FC } from 'react';
 import { useAppSelector, useAppDispatch } from 'redux/hooks';
 import { increment, decrement } from 'redux/reducers/sampleSlice';
 import type { RootState } from 'redux/store';
-import { TitleStyled } from 'Sample.styles';
+import { TitleStyled } from './Sample.styles';
 import { useTranslation } from 'react-i18next';
-import GoogleAuth from 'GoogleAuth/GoogleAuth';
+// import GoogleAuth from 'GoogleAuth/GoogleAuth';
+import GoogleAuth from 'components/GoogleAuth/GoogleAuth';
```

Comment:
why do you add anything to the sample? its just exaple

Reply:
it was wrong path and I fix it